gts-macros/README.md

```
@@ -223,61 +237,70 @@ use gts::gts::GtsSchemaId;
+    description = "Rate limit error",
+    properties = "retry_after"
 )]
-pub struct BaseEventV1<P> {
-    pub id: GtsInstanceId,     // Event topic ID field
-    pub r#type: GtsSchemaId,   // Event type (schema) ID field - ❌ Cannot have both!
+pub struct RateLimitErrorV1 {
+    pub gts_type: GtsSchemaId,  // ❌ Error! Must be in properties or removed
+    pub retry_after: u64,
+}
+```
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)